src/program/lwaftr/tests/subcommands/loadtest_test.py

```diff
@@ -8,48 +8,40 @@
 
 import unittest
 
-from lib import sh
-from lib.test_env import BENCHDATA_DIR, DATA_DIR, SNABB_CMD, nic_names
+from lib.test_env import (
+    BENCHDATA_DIR, DATA_DIR, SNABB_CMD, BaseTestCase, nic_names)
 
 
 SNABB_PCI0, SNABB_PCI1 = nic_names()
 
 
 @unittest.skipUnless(SNABB_PCI0 and SNABB_PCI1, 'NICs not configured')
-class TestLoadtest(unittest.TestCase):
+class TestLoadtest(BaseTestCase):
 
-    run_cmd_args = (
-        SNABB_CMD, 'lwaftr', 'run',
+    daemon_args = (
+        str(SNABB_CMD), 'lwaftr', 'run',
         '--bench-file', '/dev/null',
-        '--conf', DATA_DIR / 'icmp_on_fail.conf',
+        '--conf', str(DATA_DIR / 'icmp_on_fail.conf'),
         '--on-a-stick', SNABB_PCI0,
     )
-
-    loadtest_cmd_args = (
-        SNABB_CMD, 'lwaftr', 'loadtest',
+    loadtest_args = (
+        str(SNABB_CMD), 'lwaftr', 'loadtest',
         '--bench-file', '/dev/null',
         # Something quick and easy.
         '--program', 'ramp_up',
         '--step', '0.1e8',
         '--duration', '0.1',
         '--bitrate', '0.2e8',
         # Just one card for on-a-stick mode.
-        BENCHDATA_DIR / 'ipv4_and_ipv6_stick_imix.pcap', 'ALL', 'ALL', SNABB_PCI1,
+        str(BENCHDATA_DIR / 'ipv4_and_ipv6_stick_imix.pcap'), 'ALL', 'ALL',
+        SNABB_PCI1,
     )
-
-    # Use setUpClass to only setup the "run" daemon once for all tests.
-    @classmethod
-    def setUpClass(cls):
-        cls.run_cmd = sh.sudo(*cls.run_cmd_args, _bg=True)
+    wait_for_daemon_startup = True
```

Comment:
It this line meant to be in `setUpClass` body or at class level? If it's a class level I think it's more readable to move it before the method definition, together with other class properties such as `loadtest_args`, `daemon_args`, etc